fcitx-configtool:5 configure failed

configure 阶段报错
```
cmake -C /var/tmp/portage/app-i18n/fcitx-configtool-5.0.14/work/fcitx-configtool-5.0.14_build/gentoo_common_config.cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/usr -DKDE_INSTALL_USE_QT_SYS_PATHS=yes -DENABLE_KCM=yes -DENABLE_CONFIG_QT=yes -DENABLE_TEST=no -DCMAKE_BUILD_TYPE=Gentoo -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/app-i18n/fcitx-configtool-5.0.14/work/fcitx-configtool-5.0.14_build/gentoo_toolchain.cmake /var/tmp/portage/app-i18n/fcitx-configtool-5.0.14/work/fcitx5-configtool-5.0.14
loading initial cache file /var/tmp/portage/app-i18n/fcitx-configtool-5.0.14/work/fcitx-configtool-5.0.14_build/gentoo_common_config.cmake
-- The C compiler identification is GNU 11.3.0
-- The CXX compiler identification is GNU 11.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib/ccache/bin/x86_64-pc-linux-gnu-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/lib/ccache/bin/x86_64-pc-linux-gnu-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could not set up the appstream test. appstreamcli is missing.
-- Looking for GLIBC
-- Looking for GLIBC - found
-- Performing Test _OFFT_IS_64BIT
-- Performing Test _OFFT_IS_64BIT - Success
-- Performing Test HAVE_DATE_TIME
-- Performing Test HAVE_DATE_TIME - Success
-- Found Gettext: /usr/bin/msgmerge (found version "0.21") 
-- Found XKBCommon_XKBCommon: /usr/lib64/libxkbcommon.so (found version "1.4.1") 
-- Found XKBCommon: /usr/lib64/libxkbcommon.so (found version "1.4.1") found components: XKBCommon 
CMake Error at CMakeLists.txt:46 (find_package):
  By not providing "FindKF5Plasma.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "KF5Plasma", but CMake did not find one.

  Could not find a package configuration file provided by "KF5Plasma" with
  any of the following names:

    KF5PlasmaConfig.cmake
    kf5plasma-config.cmake

  Add the installation prefix of "KF5Plasma" to CMAKE_PREFIX_PATH or set
  "KF5Plasma_DIR" to a directory containing one of the above files.  If
  "KF5Plasma" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
```

我不是KDE用户，结果愣逼得我装了 kde-framework/plasma:5 才解决……这玩意是不是和KDE绑得越来越紧了别哪天整个装上一个KDE桌面才能用……

另外还有个问题，[依赖的这一行](https://github.com/microcai/gentoo-zh/blob/eaffcc98d55bba5795c7bc1c53bc951c2eb04336/app-i18n/fcitx-configtool/fcitx-configtool-5.0.14.ebuild#L47)为啥一定要写上和:4冲突呢，我感觉也没必要罢。感觉这个block除了阻挡了专门折腾共存的用户以外也没什么别的作用了。不需要共存的也不会受影响，而折腾共存的就还得自己改，感觉画蛇添足

--- a/app-i18n/fcitx-configtool/fcitx-configtool-5.0.14.ebuild
+++ b/app-i18n/fcitx-configtool/fcitx-configtool-5.0.14.ebuild
@@ -20,7 +20,7 @@ HOMEPAGE="https://fcitx-im.org/ https://github.com/fcitx/fcitx5-configtool"
 
 LICENSE="GPL-2+"
 SLOT="5"
-IUSE="+kcm +config-qt test"
+IUSE="kcm +config-qt test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="app-i18n/fcitx:5
@@ -44,7 +44,7 @@ RDEPEND="app-i18n/fcitx:5
 	config-qt? (
 		kde-frameworks/kitemviews:5
 	)
-	!${CATEGORY}/${PN}:4[-minimal(-)]"
+"
 
 DEPEND="${RDEPEND}
 	kde-frameworks/extra-cmake-modules:5

--- a/app-i18n/fcitx-configtool/fcitx-configtool-9999.ebuild
+++ b/app-i18n/fcitx-configtool/fcitx-configtool-9999.ebuild
@@ -20,7 +20,7 @@ HOMEPAGE="https://fcitx-im.org/ https://github.com/fcitx/fcitx5-configtool"
 
 LICENSE="GPL-2+"
 SLOT="5"
-IUSE="+kcm +config-qt test"
+IUSE="kcm +config-qt test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="app-i18n/fcitx:5
@@ -44,7 +44,7 @@ RDEPEND="app-i18n/fcitx:5
 	config-qt? (
 		kde-frameworks/kitemviews:5
 	)
-	!${CATEGORY}/${PN}:4[-minimal(-)]"
+"
 
 DEPEND="${RDEPEND}
 	kde-frameworks/extra-cmake-modules:5

